.github/workflows/release.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           MEMCACHE_SERVERS: "localhost:11211"
         run: |
           bundle exec rubocop
-          # bundle exec rspec
+          bundle exec rspec
```

Comment:
On the deploy workflow, instead of running just rspec it has a series of specific specfiles it executes

https://github.com/datacite/lupo/blob/9f4380dfbc19515e040771d373c33fdc88331519/.github/workflows/deploy.yml#L80

Should we do the same here?

Reply:
I did that a while ago.  I thought I was seeing flakiness in the test run due to resource limitations so I split up the run.  It may run ok in the test environment.  Give it a try.  Probably better to have just the one command.

Reply:
I think it should be ok to roll this back into one command - that's certainly how I usually run it locally and I haven't seen any issues.